tools/swift-inspect/Sources/SwiftInspectClientShims/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module SwiftInspectClientShims {
```

Comment:
These aren't really shims, they are part of the interface itself.

Reply:
Done